bundles/org.openhab.binding.broadlink/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -0,0 +1,90 @@
+
+	<channel-type id="command">
+		<item-type>String</item-type>
+		<label>IR Command</label>
+		<description>The IR command sent to the device</description>
+		<category>Text</category>
+	</channel-type>
+	<channel-type id="learning-control">
+		<item-type>String</item-type>
+		<label>Remote IR Learning Control</label>
+		<description>Instructs the Broadlink device to perform tasks related to IR code learning</description>
+		<state readOnly="false">
+			<options>
+				<option value="LEARN">Learn IR command</option>
+				<option value="CHECK">Check and save IR command</option>
+				<option value="MODIFY">Modify a previously created IR command</option>
+				<option value="DELETE">Delete IR command</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="rfcommand">
+		<item-type>String</item-type>
+		<label>RF Command</label>
+		<description>The RF command sent to the device</description>
+		<category>Text</category>
+	</channel-type>
+	<channel-type id="learning-rf-control">
+		<item-type>String</item-type>
+		<label>Remote RF Learning Control</label>
+		<description>Instructs the Broadlink device to perform tasks related to RF code learning</description>
+		<state readOnly="false">
+			<options>
+				<option value="LEARN">Learn RF command</option>
+				<option value="CHECK">Check and save RF command</option>
+				<option value="MODIFY">Modify a previously created RF command</option>
+				<option value="DELETE">Delete RF command</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="temperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature</label>
+		<category>Temperature</category>
+		<state pattern="%.1f %unit%" readOnly="true"/>
+	</channel-type>
+	<channel-type id="humidity">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Humidity</label>
+		<category>Humidity</category>
+		<state readOnly="true" min="0" max="100" pattern="%.2f %unit%"/>
+	</channel-type>
+	<channel-type id="power-on">
+		<item-type>Switch</item-type>
+		<label>Socket Power</label>
+		<category>PowerOutlet</category>
+	</channel-type>
+	<channel-type id="night-light">
+		<item-type>Switch</item-type>
+		<label>Night Light</label>
+		<category>Lightbulb</category>
+	</channel-type>
+	<channel-type id="power-consumption" advanced="true">
+		<item-type>Number:Power</item-type>
+		<label>Power Consumption</label>
+		<state pattern="%.2f" readOnly="true"></state>
+	</channel-type>
+	<channel-type id="light">
+		<item-type>String</item-type>
+		<label>Illumination</label>
+		<description>Light conditions</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="noise">
+		<item-type>String</item-type>
+		<label>Background Noise</label>
+		<description>Noise conditions</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="air">
+		<item-type>String</item-type>
```

Comment:
Is this a constant set of known strings? If so it would be nice to have those listed in an option tag. (same for other string channel id's)

Reply:
I have no idea, as I do not own these devices.